runner/proto/results.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+package proto;
+option go_package = "./proto";
+
+import "runner/proto/oscal.proto";
+
+message ResultRequest { AssessmentResult Result = 1; }
+
+message ResultResponse {}
+
+service AddHelper { rpc AddResult(ResultRequest) returns (ResultResponse); }
```

Comment:
`ApiHelper`

Reply:
I also think `CreateResult` would be more explicit. The plugin doesn't understand the implementation details. Their intention is to create a result. Calling it CreateResult leaves no nuance as to what is going to happen when they call this method.